pkg/verifiedcar/verifiedcar.go

```diff
@@ -72,14 +77,28 @@ func (cfg Config) Verify(ctx context.Context, rdr io.Reader, lsys linking.LinkSy
 	if len(cbr.Roots) != 1 || cbr.Roots[0] != cfg.Root {
 		return 0, 0, ErrBadRoots
 	}
+	return cfg.VerifyBlockStream(ctx, cbr, lsys)
+}
+
+type BlockReader interface {
```

Comment:
maybe move this to near the top of the file, I think we're fairly consistent with that as a pattern?